wagtail/images/shortcuts.py

```diff
@@ -21,3 +21,26 @@ def get_rendition_or_not_found(image, specs):
         rendition = Rendition(image=image, width=0, height=0)
         rendition.file.name = "not-found"
         return rendition
+
+def get_multiple_renditions_or_not_found(image, specs):
```

Comment:
For consistency with the above function,

```suggestion
def get_renditions_or_not_found(image, specs):
```